ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt

```diff
@@ -169,7 +169,7 @@ hideall "--sync--"
 705.7 "J Storm + Waves x16" Ability { id: "4876", source: "Brute Justice" } duration 50
 731.7 "Eternal Darkness Enrage" Ability { id: "4875", source: "Cruise Chaser" }
 771.9 "Divine Judgment Enrage" Ability { id: "4879", source: "Alexander Prime" } window 67,5
-787.3 "Divine Judgment" Ability { id: "487A", source: "Alexander" }
+787.3 "Divine Judgment" Ability { id: "487A", source: "Alexander" } window 80,10
```

Comment:
If waiting until the last possible second to kill AP can push `Divine Judgment` back to something more like `795.3`, would it make sense just to change the timeline entry to `795.3` and then do a `window 90,2`? 

Since the ability timing is variable, the exact time listed on the line is I guess a bit arbitrary, but it will control when the ability shows up on the timeline by default until there's a sync.  In that case, I think it might make more sense for the default position to be at the latest possible execution time, and then let it sync if it happens early.  Otherwise, you would end up with the ability running on the timeline and disappearing while the phase is still active (even though it will later sync), which seems less optimal.

Reply:
I don't have exact timings for everything and my team's done with TEA for now, so I could only submit this adjustment because it's the only one I was actually able to test.

If we bump the start time up for `Divine Judgement`, then the following `Down for the Count` and `--targetable--` entries also need to be bumped up, and I'm not sure if the linear time increase of `+8s` would always be applicable or not. We ***always*** kill extremely close to the cast finishing, so if e.g. a speedkill run were attempted, I'm not sure if those times would still line up.

I can adjust the times for those as part of this PR as requested, but I don't have any way to actually test those changes beyond the one log file I still have ready for testing.

Reply:
> If we bump the start time up for `Divine Judgement`, then the following `Down for the Count` and `--targetable--` entries also need to be bumped up, and I'm not sure if the linear time increase of `+8s` would always be applicable or not. We _**always**_ kill extremely close to the cast finishing, so if e.g. a speedkill run were attempted, I'm not sure if those times would still line up.
> 
> I can adjust the times for those as part of this PR as requested, but I don't have any way to actually test those changes beyond the one log file I still have ready for testing.

You could [test against log files from fflogs](https://github.com/OverlayPlugin/cactbot/blob/main/docs/TimelineGuide.md#using-test_timelinets-with-fflogs).  Perhaps grab a handful of [recent-ish speedkills](https://www.fflogs.com/zone/rankings/43#boss=1062)?

Reply:
I tend to agree with wexxlee that we want the display time for an ability like this to be the absolute latest that's possible. I don't know of any other situation where the timing of mechanics that follow a synced cast like that would vary, so I think the adjustment should be safe enough, particularly if we can get some confirmation from testing against FFLogs.